kolibri/plugins/learn/assets/src/views/LibraryPage/OtherLibraries.vue

```diff
@@ -257,4 +286,67 @@
     margin-left: 0.75em;
   }
 
+  @media screen and (max-width: 600px) {
```

Comment:
`useKResponsiveWindow` is used for styling based on reactive window size information rather than CSS Media queries. For more detailed breakpoint styling, using `KGridItem` along with `useKResponsiveWindow` could be useful. There’s some examples of this being done in the Kolibri codebase [here](https://github.com/yashhash2/kolibri/blob/48eaf0366de77bb480dd5fce726433932845707d/kolibri/plugins/facility/assets/src/views/DataPage/index.vue#L109), [here](https://github.com/yashhash2/kolibri/blob/my-awesome-changes/kolibri/plugins/device/assets/src/views/HeaderWithOptions.vue#L15), and [here](https://github.com/yashhash2/kolibri/blob/48eaf0366de77bb480dd5fce726433932845707d/kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/QuestionListPreview.vue#L5).

I saw that you mentioned logo overlapping with text on some screen sizes; perhaps this issue could be fixed by using conditional styling?